src/main/java/org/rostekus/hint/HintController.java

```diff
@@ -0,0 +1,41 @@
+package org.rostekus.hint;
+
+import org.rostekus.game.Game;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping(path = "api/v1/hints")
+public class HintController {
+    private final HintService hintService;
+
+    @Autowired HintController(HintService hintService) { this.hintService = hintService; }
+
+    @GetMapping
+    public List<Hint> getHint() { return hintService.getAllHints(); }
+
+    @PostMapping
+    @ResponseStatus(code = HttpStatus.CREATED)
+    public void postHint(@RequestBody Hint h, @PathVariable UUID id) {
+        Game g = hintService.gameRepository.findById(id).orElse(null);
+        hintService.addNewHint(h, g.getId());
+    }
```

Comment:
@PathVariable oznacza że w url będzie podane ID a tego nie potrzebujemy bo ID dla hinta zostanie automatycznie dla nas wygenerowane

Reply:
Do tego jak chcemy korzystać z @PathVariable to robimy coś takiego
```
@GetMapping("/{id}") <---
    @ResponseStatus(code = HttpStatus.OK)
    public Game getGameById(@PathVariable UUID id)
```
czyli mówimy gdzie ta zmienna o nazwie Id będzie w url